<launch>
  <node  pkg="mono_cam_calib" name="mono_cam_calib" type="mono_cam_calib" output="screen">

  	<param name = "dx" type="double" value="61.5"/>
  	<param name = "dy" type="double" value="61.5"/>
  	<param name = "checker_x_number" type = "int" value = "9" />
  	<param name = "checker_y_number" type = "int" value = "6" />
    <param name = "result_file_intrinsic" type="string" value="$(find mono_cam_calib)/result/"/>
		<param name = "result_file_RT" type="string" value="$(find mono_cam_calib)/result/"/>
		<param name = "view_num" type="int" value="10"/>
  	<param name = "image_sub_name" type="string" value="/pylon_camera_node/image_raw"/>
    <param name = "Intrinsic_debug" type="bool" value="true"/>
    <param name = "RT_debug" type="bool" value="false"/>

  </node>
</launch>
